### ISSUE TITLE
Is it possible to do more broad authorization rules?

### ISSUE THREAD

**Issue by fyem** [mbonneau/ThruwayAuthorizationExample#1]
I would like to do rules that are more broad than the ones in the example.

Something like any user can subscribe on any URI but nobody may publish (except the server itself).

How would I create an authorization rule like that?

**Comment by mbonneau** [comment]
@fyem - Not sure how much control you need, but you could disallow all publishing and then use an internal client or add a trusted transport that the "server" uses to publish on.

**Comment by fyem** [comment]
@mbonneau yes, that sounds exactly what I would want to do. How would my authorizationrule have to look like to disallow all publishing?

**Comment by mbonneau** [comment]
I explain some use of authorization here: http://voryx.net/authorization-permissions-with-thruway/
